app/ui/client/components/header/headerRoom.js

```diff
@@ -28,6 +28,11 @@ const getUserStatus = (id) => {
 	return roomTypes.getUserStatus(roomData.t, id) || 'offline';
 };
 
+const getUserStatusText = (id) => {
+	const roomData = Session.get(`roomData${ id }`);
+	return roomTypes.getUserStatusText(roomData.t, id) || 'offline';
```

Comment:
This will cause the status to be shown as offline if there's no custom message.

Reply:
Weird... My code doesn't have this...
![image](https://user-images.githubusercontent.com/6295044/60024462-cc012080-9665-11e9-9d04-1fdd0315ec77.png)

Shows commmited, too.

Reply:
https://github.com/RocketChat/Rocket.Chat/pull/14853/commits/31137ac1907a2d9cb59bf66019abe8cb73f456ef#diff-a89db19a330b54652f42b8fa6cc29bcdR32

Reply:
Sorry, my mistake. I think I was looking at a specific commit when I commented.